controller/index.js

```diff
@@ -1,15 +1,58 @@
-const accountSid = process.env.TWILIO_ACCOUNT_SID;
-const authToken = process.env.TWILIO_AUTH_TOKEN;
-const client = require('twilio')(accountSid, authToken);
+const config = require('../config/index');
+const twilio = require('twilio');
+const statusCode = require('http-status');
+const fetch = require('node-fetch');
+const client = twilio(config.ACCOUNT_SID, config.AUTH_TOKEN);
 
 exports.sendSms = async (req, res) => {
-    try {
-        
-    } catch (error) {
-        
-    }
-};
+	const { message, mobile_num } = req.body;
+	if (!message || message == ' ') {
+		return res.status(statusCode.PRECONDITION_FAILED).json({
+			message: 'message field is empty',
+		});
+	}
+	if (!mobile_num || mobile_num == ' ') {
+		return res.status(statusCode.PRECONDITION_FAILED).json({
+			message: 'mobile number field is empty',
+		});
+	}
 
-exports.checkBalance = (req, res) => {
+	try {
+		smsOptions = {
+			body: message,
+			from: config.TWILIO_NUMBER,
+			to: mobile_num,
+		};
+		const sendSms = await client.messages.create(smsOptions);
+		if (sendSms) {
+			return res.status(statusCode.OK).json({
+				message: 'message sent successfully',
+				sendSms,
+			});
+		}
+	} catch (error) {
+		console.log(`error in sending sms >>> ${error}`);
+		return res.status(statusCode.SERVICE_UNAVAILABLE).json({
+			message: 'error in sending sms.. try again later',
+		});
+	}
+};
 
-};
\ No newline at end of file
+exports.checkBalance = async (req, res) => {
+	const balanceUrl = `https://api.twilio.com/2010-04-01/Accounts/${config.ACCOUNT_SID}/Balance.json`;
+	try {
+		const response = await fetch(balanceUrl);
+		const data = await response.json();
+		console.log(data);
+		return res.status(statusCode.OK).json({
+			message: 'Received sms balance',
+			balance: data.balance,
+			currency: data.currency
+		})
```

Comment:
I think the date has to be the recent date

Reply:
That's the default date that should be used..
It's just to add our own auth token and account sid to the url

Reply:
Okay.. I already solved it.

can I just make a PR for you to see?